### ISSUE TITLE
[source-mysql] Possible wrong type conversion

### ISSUE THREAD

**Issue by theyueli** [airbytehq/airbyte#41282]
@michel-tricot observed an issue reported on reddit:

https://www.reddit.com/r/dataengineering/comments/1dz1onq/weird_mysql_cdc_binlog_behavior/

**Comment by rodireich** [comment]
Enum types are converted to a String by debezium.
It's worth checking perhaps if the initial sync matches in terms of mapped value.

**Comment by rodireich** [comment]
It does, it [appears](https://github.com/airbytehq/airbyte/blob/master/airbyte-integrations/connectors/source-mysql/src/main/java/io/airbyte/integrations/source/mysql/MySqlSourceOperations.java#L157)
```
case CHAR, VARCHAR, TINYTEXT, TEXT, MEDIUMTEXT, LONGTEXT, ENUM, SET -> setString(preparedStatement, parameterIndex, value);
```

**Comment by rodireich** [comment]
https://issues.redhat.com/browse/DBZ-7792
This is the bug in debezium I think - fixed in 2.7.?

**Comment by evantahler** [comment]
Closing issue as we will upgrade mysql dbz in Q3A
